tests/nodes/PhysicsBody/tests/move_and_collide/3d/collide_sphere_capsule.gd

```diff
@@ -0,0 +1,123 @@
+extends PhysicsUnitTest3D
+
+
+# Tolerances
+const POSITION_TOLERANCE := 0.01
+const NORMAL_TOLERANCE := 0.01
+
+
+var simulation_duration := 20
+var test_failed := false
+
+func test_description() -> String:
+	return """Checks whether the move_and_collide between sphere and capsule works
+	correctly."""
+	
+func test_name() -> String:
+	return "PhysicsBody3D | testing collide sphere with capsule"
+
+
+var tested_body: CharacterBody3D
+var static_body: StaticBody3D
+
+func start() -> void:
+	tested_body = $Tested
+	static_body = $Static
+	
+	var test_lambda: Callable = func(p_target, p_monitor):
```

Comment:
It's better to type the monitor: `var test_lambda: Callable = func(p_target, p_monitor: GenericManualMonitor):`

Reply:
Modified to have the correct typing on the monitor.